app/dashboard/sync/celo.py

```diff
@@ -63,6 +63,5 @@ def sync_celo_payout(fulfillment):
             fulfillment.payout_status = 'done'
             fulfillment.accepted_on = timezone.now()
             fulfillment.accepted = True
+            fulfillment.save()
             record_payout_activity(fulfillment)
-
-        fulfillment.save()
```

Comment:
why was this removed ?
This means L58 would not get saved if the txn is in pending

Reply:
i noticed it was same way on etc and eth... should i revert this? @thelostone-mc

Reply:
yup ! I'd say add it to ETC as well